src/wp-admin/includes/class-wp-site-health.php

```diff
@@ -1656,6 +1656,120 @@ public function get_test_https_status() {
 		return $result;
 	}
 
+	/**
+	 * Tests if Full Page Cache is available.
+	 *
+	 * @since 6.1.0
+	 *
+	 * @return array The test results.
```

Comment:
Nit-pick, but let's align with other Site Health test methods in core:
```suggestion
	 * @return array The test result.
```

Reply:
@felixarntz I actually see a mix of them (result/results) also in core.

Reply:
Fair point. I'd say let's still go for the singular since one check arguably gives only one result :)